component/loki/source/windowsevent/component.go

```diff
@@ -0,0 +1,37 @@
+//go:build !windows
+
+package windowsevent
+
+import (
+	"context"
+
+	"github.com/grafana/agent/component"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name: "loki.source.windowsevent",
+		Args: Arguments{},
+
+		Build: func(opts component.Options, args component.Arguments) (component.Component, error) {
+			return &FakeComponent{}, nil
+		},
+	})
+}
+
+var (
+	_ component.Component = (*FakeComponent)(nil)
+)
+
+// FakeComponent implements the loki.source.windowsevent component for non-windows environments.
+type FakeComponent struct {
+}
+
+func (f *FakeComponent) Run(ctx context.Context) error {
+	<-ctx.Done()
```

Comment:
Should this log something to tell the user that the component won't do anything?